src/metabase/driver/crate/native.clj

```diff
@@ -0,0 +1,31 @@
+(ns metabase.driver.crate.native
+  (:require [clojure.java.jdbc :as jdbc]
+            [metabase.models.database :refer [Database]]
+            [metabase.db :refer [sel]]
+            [metabase.driver.generic-sql :as sql]
+            [clojure.tools.logging :as log]
+            [metabase.util :as u]
+            [metabase.driver.generic-sql.native :as n]))
+
+(defn process-and-run
```

Comment:
Crate does not support transactions. Therefore we need to use the the `jdbc/with-db-connection` instead of `jdbc/with-db-transaction` which sets autocommit to `false` - which is not good for us 😢 since our JDBC driver is returning a `SQLFeatureNotSupportedException` https://github.com/crate/crate-jdbc/blob/1.10/src/main/java/io/crate/client/jdbc/CrateConnection.java#L90.